extensions-core/namespace-lookup/src/main/java/io/druid/server/namespace/URIExtractionNamespaceFunctionFactory.java

```diff
@@ -151,9 +151,23 @@ public String call()
             versionRegex = null;
           }
         } else {
-          final Path filePath = Paths.get(extractionNamespace.getUri());
+          final URI rawURI = extractionNamespace.getUri();
+          final Path filePath = Paths.get(rawURI.getPath());
```

Comment:
or calling `puller.getLatestVersion`; that seems unnecessary when we provide a specific uri.